tests/platform/windows/windows-kurzweil-testSpec.txt

```diff
@@ -0,0 +1,10 @@
+windows-kurzweil-testSpec.js
+
+Descriptions:
+This will run the acceptance tests for Kurzweil 3000 (KESI). It uses 2 NP sets: kurzweil_common.json and kurzweil_application.json
+
+Prerequisites:
+* Kurzweil 3000 needs to be installed
+* Need to set up username "gpii" for Kurzweil(incomplete)
```

Comment:
What does (incomplete) refer to here?

It seems that in order to make this integration more useful, we would need to in addition expose a preference for the Kurzweil username in the user's NP set.

Is there any automated way that we can find to provision the gpii user - can it be done just via the filesystem or is some interaction with the app required?